.circleci/config.yml

```diff
@@ -263,9 +252,27 @@ jobs:
           paths:
             - dist
             - node_modules
-  deploy:
+ 
+  deploy_etl:
+    environment:
+      - VERSION: "0.1.3"
+    working_directory: ~/repo/extract_endpoint
```

Comment:
this is just deploying the endpoint code I think, not the etl code? If so the section should be called `deploy_endpoint`.